src/system.jl

```diff
@@ -99,6 +99,60 @@ struct Branch
     break_points::Tuple{Float64, Float64}
     "Price penalties for each of the break points of the branch (\$)"
     penalties::Tuple{Float64, Float64}
+    "Resistance of the transformer (Ohm)"
+    resistance::Float64
+    "Reactance of the transformer or branch (Ohm)"
```

Comment:
What do we want the units of `Resistance` and `Reactance` to be?

Reply:
As mentioned in the discussion we had in the FNDP PR, I think having all branch parameters in pu is the best course unfortunately. It just makes everything much simpler, otherwise we'd need to rely on conversions whenever doing operations with parameters from different branches.